modules/cnf-topology-aware-lifecycle-manager-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-topology-aware-lifecycle-manager-operator-troubleshooting_{context}"]
+= Troubleshooting missed Operator updates due to out-of-date policy compliance states
+
+In some scenarios, TALM might miss Operator updates due to an out-of-date policy compliance state.
```

Comment:
s/TALM/{cgu-operator-first}

Should this be _the_ TALM/{cgu-operator-first}? Seems inconsistent in the docs.